### ISSUE TITLE
correct td command quotation mark position

### ISSUE THREAD

**Issue by superewald** [rizinorg/book#96]
The example of loading types from string at [Loading Types](https://book.rizin.re/analysis/types.html#loading-types) seems to have wrong quotation marks.

`[0x000051c0]> "td struct foo {char* a; int b;}"` does nothing for me (using rizin 0.5.0 @ windows-x86-64).
`[0x000051c0]> td "struct foo {char* a; int b;}"` works (note the quotation mark being moved after td command).

**Comment by XVilka** [comment]
Yes, you are right, it's a consequence of transitioning to the new command parser:
- https://rizin.re/posts/rzshell/
- https://github.com/rizinorg/rizin/issues/1342
